system-addon/test/functional/mochitest/browser_topsites_section.js

```diff
@@ -23,13 +28,15 @@ test_newtab({
     await ContentTaskUtils.waitForCondition(() => content.document.querySelector(".top-site-icon"),
       "Topsite tippytop icon not found");
     // There are only topsites on the page, the selector with find the first topsite menu button.
-    const topsiteContextBtn = content.document.querySelector(".top-sites-list .context-menu-button");
+    let topsiteContextBtn = content.document.querySelector(".top-sites-list .context-menu-button");
     topsiteContextBtn.click();
 
-    const contextMenu = content.document.querySelector(".top-sites-list .context-menu");
-    ok(contextMenu && !contextMenu.hidden, "Should find a visible topsite context menu");
+    await new Promise(resolve => setTimeout(resolve, 0));
```

Comment:
Probably better to use a ```waitForCondition``` statement here, to keep it consistent with the rest of the test:
```js
await ContentTaskUtils.waitForCondition(() => content.document.querySelector(".top-sites-list .context-menu"), 
   "No context menu found");
```